src/targets/gpu/mlir.cpp

```diff
@@ -622,12 +669,21 @@ struct mlir_program
         mlir_pass_manager pm_back{mlirPassManagerCreate(ctx.get())};
         // 1st pipeline to call
         mlirMIGraphXAddHighLevelPipeline(pm_front.get());
-        mlirPassManagerRunOnOp(pm_front.get(), mlirModuleGetOperation(mmodule.get()));
```

Comment:
Since these are refactored into seperate functions in #1965, I would prefer that PR is merged first before this one.

Reply:
1965 has been merged